src/main/java/ru/practicum/shareit/user/controller/UserController.java

```diff
@@ -17,36 +23,38 @@
 @RequestMapping(path = "/users")
 @RequiredArgsConstructor
 public class UserController {
-    private final UserServiceImpl userService;
 
-    @GetMapping
-    public ResponseEntity<List<UserDto>> getAll() {
-        log.info("Получен запрос на получение всех пользователей");
-        return new ResponseEntity<>(userService.getAll(), HttpStatus.OK);
+    private final UserService userService;
+
+    @PostMapping
+    public ResponseEntity<UserDto> add(@Valid @RequestBody UserDto userDto) {
+        log.info("Получен запрос на добавление пользователя");
+        return new ResponseEntity<>(userService.add(userDto), HttpStatus.CREATED);
     }
 
-    @GetMapping("/{id}")
-    public ResponseEntity<UserDto> getById(@PathVariable Long id) {
-        log.info("Получен запрос на получение пользователя id-{}", id);
-        return new ResponseEntity<>(userService.getById(id), HttpStatus.OK);
+    @GetMapping
+    public ResponseEntity<List<UserDto>> getAllUsers() {
+        log.info("Получен запрос на получение всех пользователей");
+        return new ResponseEntity<>(userService.getAllUsers(), HttpStatus.OK);
     }
 
-    @PostMapping
-    public ResponseEntity<UserDto> create(@Valid @RequestBody UserDto userDto) {
-        log.info("Получен запрос на добавление пользователя");
-        return new ResponseEntity<>(userService.create(toUser(userDto)), HttpStatus.CREATED);
+    @GetMapping(value = "/{id}")
+    public ResponseEntity<UserDto> getUserById(@PathVariable long id)  {
+        log.info(String.format("Получен запрос на получение пользователя id-{}", id));
```

Comment:
`String.format` - лишний, так как параметры в логирование передаются через placeholder `{}`
```suggestion
        log.info("Получен запрос на получение пользователя id-{}", id);
```